examples/cookie-game/src/state.ts

```diff
@@ -0,0 +1,692 @@
+    const mineCps = cookieBaker.numberOfMine * st.init_mineCps;
+    const factoryCps = cookieBaker.numberOfFactory * st.init_factoryCps;
+    const bankCps = cookieBaker.numberOfBank * st.init_bankCps;
+    const templeCps = cookieBaker.numberOfTemple * st.init_templeCps;
+    const wizardCps = cookieBaker.numberOfWizard * st.init_wizardCps;
+    const shipmentCps = cookieBaker.numberOfShipment * st.init_shipmentCps;
+    const alchemyCps = cookieBaker.numberOfAlchemy * st.init_alchemyCps;
+    const portalCps = cookieBaker.numberOfPortal * st.init_portalCps;
+    const timeMachineCps = cookieBaker.numberOfTimeMachine * st.init_timeMachineCps;
+    const antimatterCps = cookieBaker.numberOfAntimatter * st.init_antimatterCps;
+    const prismCps = cookieBaker.numberOfPrism * st.init_prismCps;
+    const chanceMakerCps = cookieBaker.numberOfChanceMaker * st.init_chanceMakerCps;
+    const fractalCps = cookieBaker.numberOfFractal * st.init_fractalCps;
+    const javaScriptCps = cookieBaker.numberOfJavaScript * st.init_javascriptCps;
+    const idleverseCps = cookieBaker.numberOfIdleverse * st.init_idleverseCps;
+    const cordexCps = cookieBaker.numberOfCordex * st.init_cordexCps;
+
+    cookieBaker.cursorCost = cursorCost;
+    cookieBaker.grandmaCost = grandmaCost;
+    cookieBaker.farmCost = farmCost;
+    cookieBaker.mineCost = mineCost;
+    cookieBaker.factoryCost = factoryCost;
+    cookieBaker.bankCost = bankCost;
+    cookieBaker.templeCost = templeCost;
+    cookieBaker.wizardCost = wizardCost;
+    cookieBaker.shipmentCost = shipmentCost;
+    cookieBaker.alchemyCost = alchemyCost;
+    cookieBaker.portalCost = portalCost;
+    cookieBaker.timeMachineCost = timeMachineCost;
+    cookieBaker.antimatterCost = antimatterCost;
+    cookieBaker.prismCost = prismCost;
+    cookieBaker.chanceMakerCost = chanceMakerCost;
+    cookieBaker.fractalCost = fractalCost;
+    cookieBaker.javaScriptCost = javaScriptCost;
+    cookieBaker.idleverseCost = idleverseCost;
+    cookieBaker.cordexCost = cordexCost;
+
+    cookieBaker.cursorCps = cursorCps;
+    cookieBaker.grandmaCps = grandmaCps;
+    cookieBaker.farmCps = farmCps;
+    cookieBaker.mineCps = mineCps;
+    cookieBaker.factoryCps = factoryCps;
+    cookieBaker.bankCps = bankCps;
+    cookieBaker.templeCps = templeCps;
+    cookieBaker.wizardCps = wizardCps;
+    cookieBaker.shipmentCps = shipmentCps;
+    cookieBaker.alchemyCps = alchemyCps;
+    cookieBaker.portalCps = portalCps;
+    cookieBaker.timeMachineCps = timeMachineCps;
+    cookieBaker.antimatterCps = antimatterCps;
+    cookieBaker.prismCps = prismCps;
+    cookieBaker.chanceMakerCps = chanceMakerCps;
+    cookieBaker.fractalCps = fractalCps;
+    cookieBaker.javaScriptCps = javaScriptCps;
+    cookieBaker.idleverseCps = idleverseCps;
+    cookieBaker.cordexCps = cordexCps;
+    return cookieBaker;
+}
+
+export const calculateCost = (action: actions, cookieBaker: st.cookieBakerType): number => {
```

Comment:
Should this function be there? Or something else? No strong opinion, what do you think?

Reply:
I think for now, this function can be in here, but maybe later if we have more calculation function then we can move it to `utils` or something like that.